test/e2e/demos_test.go

```diff
@@ -39,6 +39,13 @@ var _ = Describe("Introduction", func() {
 		kubectlAndCheck("wait", "nncp", "vlan100", "--for", "condition=Available", "--timeout", "2m")
 	}
 
+	// Policies are not deleted as a part of the tutorial, so we need additional function here
+	cleanupConfiguration := func() {
+		deletePolicy("vlan100")
+		updateDesiredState(interfaceAbsent("eth1.100"))
+		waitForAvailableTestPolicy()
```

Comment:
since we use the TestPolicy to remove the interface we have to call after this 
resetDesiredStateForNodes()

Reply:
I don't get this. Here we use test policy to remove the VLAN, we wait until it succeeds. Good. Then we use the same policy to cleanup NICs, I don't see where is the collision

Reply:
So tutorals play around with eth1 and eth2 and that change it states so we fail on 
```yaml
/tmp/knmstate/kubernetes-nmstate/test/e2e/main_test.go:92
Expected
    <string>: {
      "cni0": "up",
      "docker0": "up",
      "eth0": "up",
      "eth1": "down",
      "eth2": "down",
      "lo": "down"
    }
to match JSON of
    <string>: {
      "cni0": "up",
      "docker0": "up",
      "eth0": "up",
      "eth1": "up",
      "eth2": "up",
      "lo": "down"
    }
```
That's why we decide on add put secondary nics down after each test if we don't really care about sideffect of using them.